platformio.ini

```diff
@@ -0,0 +1,47 @@
+; PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[platformio]
+src_dir = ./HeishaMon/
+
+[env:d1_mini]
+platform = espressif8266
+board = d1_mini
+board_build.f_cpu=160000000L
+framework = arduino
+monitor_speed = 115200
+lib_deps = 
+	knolleary/PubSubClient@^2.8
+	links2004/WebSockets@^2.4.0
+	bblanchon/ArduinoJson@^6.19.4
+	milesburton/DallasTemperature@^3.9.0
+	jenscski/DoubleResetDetect@^1.0.0
+build_flags =
+	-DMMU_IRAM_SIZE=0x8000
+	-DMMU_ICACHE_SIZE=0x4000
+	-DMMU_SEC_HEAP=0x40108000
+	-DMMU_SEC_HEAP_SIZE=0x4000
+
+[env:nodemcu]
+platform = espressif8266
+board = nodemcu
+framework = arduino
+monitor_speed = 115200
+lib_deps = 
+	knolleary/PubSubClient@^2.8
+	links2004/WebSockets@^2.4.0
+	bblanchon/ArduinoJson@^6.19.4
+	milesburton/DallasTemperature@^3.9.0
+	jenscski/DoubleResetDetect@^1.0.0
+build_flags =
+	-DMMU_IRAM_SIZE=0x8000
+	-DMMU_ICACHE_SIZE=0x4000
+	-DMMU_SEC_HEAP=0x40108000
+	-DMMU_SEC_HEAP_SIZE=0x4000
```

Comment:
Also, acording [platformio](https://docs.platformio.org/en/stable/platforms/espressif8266.html#mmu-adjusting-icache-to-iram-ratio) documentation, this should be enought
```
build_flags =
	-D PIO_FRAMEWORK_ARDUINO_MMU_CACHE16_IRAM32_SECHEAP_NOTSHARED
```